perf.groovy

```diff
@@ -639,13 +651,11 @@ parallel(
 
                     if (isPR) {
                         def opts = ""
-                        if (opt_level == 'min_opts')
-                        {
+                        if (opt_level == 'min_opts') {
```

Comment:
@brianrob Generally the same as java, which typically uses { on the same line as the if/loop/etc.